sample-app/src/main/java/com/joinforage/android/example/ui/pos/ui/ScreenWithBottomRow.kt

```diff
@@ -0,0 +1,35 @@
+package com.joinforage.android.example.ui.pos.ui
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+
+@Composable
+inline fun ScreenWithBottomRow(
+    mainContent: @Composable ColumnScope.() -> Unit,
+    bottomRowContent: @Composable RowScope.() -> Unit
```

Comment:
love me some HOCs!!

Reply:
yeah! eventually I'll use this on all the other ones I already created manually, but this PR was already big enough lol